automated/linux/ota-apps/update-apps.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
```

Comment:
This file mixes tab and spaces can that be fixed?

Reply:
fixed